core/tests/build_sources/templates/base.html

```diff
@@ -68,151 +32,21 @@
-          <div id="wrapper">
-            <div class="oppia-main-body">
-              <nav class="navbar navbar-default oppia-navbar oppia-prevent-selection" role="navigation" headroom tolerance="0" offset="0">
-                <div class="navbar-container">
-                  <top-navigation-bar></top-navigation-bar>
-                  <div class="collapse navbar-collapse oppia-navbar-collapse ng-cloak">
-                    {% block navbar_breadcrumb %}
-                    {% endblock navbar_breadcrumb %}
-
-                    {% block local_top_nav_options %}
-                    {% endblock local_top_nav_options %}
-                  </div>
-                </div>
-              </nav>
-
-              <div class="oppia-top-of-page-padding">
-              </div>
-
-              {{ warnings_and_loader() }}
-            </div>
-
-            <noscript>
-              <div class="oppia-page-cards-container">
-                <div class="md-default-theme oppia-page-card oppia-long-text">
-                  <!-- Note to developers: We replicate the translated text inline because, without JavaScript enabled, the translation engine doesn't kick in.-->
-                  <h2>
-                    <span translate="I18N_SPLASH_JAVASCRIPT_ERROR_TITLE">We Need JavaScript in Your Browser</span>
-                    <i class="material-icons">&#xE811;</i>
-                  </h2>
-                  <p translate="I18N_SPLASH_JAVASCRIPT_ERROR_DESCRIPTION"
-                     translate-values="{hrefUrl: 'http://www.enable-javascript.com/'}">
-                     Oppia is a free, open-source learning platform full of interactive activities called 'explorations'.  Sadly, Oppia requires JavaScript to be enabled in your web browser in order to function properly and your web browser has JavaScript disabled.  If you need help enabling JavaScript, <a href="http://www.enable-javascript.com">click here.</a>
-                  </p>
-                  <p translate="I18N_SPLASH_JAVASCRIPT_ERROR_THANKS">Thank you.</p>
-                </div>
-              </div>
-            </noscript>
-
-            <side-navigation-bar></side-navigation-bar>
-          </div>
-        </div>
-      </div>
-
-      <div ng-if="DEV_MODE" class="oppia-dev-mode" ng-cloak>
-        Dev Mode
-      </div>
-
-      <a ng-if="siteFeedbackFormUrl" ng-href="<[siteFeedbackFormUrl]>" target="_blank" rel="noopener"
-         class="oppia-site-feedback oppia-transition-200">
-        <i class="material-icons md-18">&#xE87F;</i>
-        <span translate="I18N_SPLASH_SITE_FEEDBACK"></span>
-      </a>
-    </div>
+    <base-content>
+      <navbar-breadcrumb></navbar-breadcrumb>
+      <content></content>
+      <page-footer></page-footer>
+    </base-content>
 
     @require('pages/footer_js_libs.html')
-
-    <script src="/templates/dev/head/App.js"></script>
-    <script src="/templates/dev/head/I18nFooter.js"></script>
-
     <script src="/templates/dev/head/pages/Base.js"></script>
```

Comment:
Remove this?

Reply:
It is used in the test

Reply:
How? And why can!t we include it by webpack?

Reply:
Since it's just one script import, I didn't think it would be necessary to include this via webpack.